features/FEATURE_BLE/targets/TARGET_Maxim/MaximBLE.cpp

```diff
@@ -52,29 +52,38 @@
 #include "hci_vs.h"
 
 /* Number of WSF buffer pools */
-#define WSF_BUF_POOLS               4
+#define WSF_BUF_POOLS 5
 
 /*! Free memory for pool buffers. */
-static uint8_t mainBufMem[768];
+static uint8_t mainBufMem[1040];
 
 /*! Default pool descriptor. */
 static wsfBufPoolDesc_t mainPoolDesc[WSF_BUF_POOLS] =
 {
   {  16,  8 },
   {  32,  4 },
   {  64,  2 },
-  { 128,  2 }
+  { 128,  2 },
+  { 272,  1 }
 };
 
 /*! WSF handler ID */
 wsfHandlerId_t maximHandlerId;
 static volatile int reset_complete;
 
+#ifdef BLE_HCI_UART
```

Comment:
How does this work with the internal HCI driver ? 
Is it possible to map the driver to other pins ?